schedule/btrfs_ipmi.yaml

```diff
@@ -0,0 +1,27 @@
+name:           btrfs@64bit-ipmi
+description:    >
+      Test installation with explicit selection of btrfs filesystem during installation. Tests installation only.
+      Maintainers: okurz
```

Comment:
Why the maintainer is okurz?

Reply:
I guess he has introduced the test suite. I have just copied the text description from openqa test suites :)

Reply:
I am/was kind of the "fallback maintainer". Feel free for anyone from you in the QSF-y team to take over